src/js/pages/MyProfile/components/Trades.jsx

```diff
@@ -50,7 +60,7 @@ class Trades extends Component {
                 {isBuyer ? 'Buy' : 'Sell' } {formatBalance(trade.tokenAmount)} {trade.token.symbol}
               </Col>
               <Col className="align-self-center text-center text-success" xs="4">
-                <span className={"p-1 text-uppercase d-inline text-white rounded-sm " + tradeStyle.className}>{tradeStyle.text}</span>
+                <span className={"p-1 text-uppercase d-inline text-white rounded-sm " + tradeStyle.className}><small>{tradeStyle.text}</small></span>
```

Comment:
I added the class `.text-small` or something like that. It puts the text in 13px. Is that what your are searching for with the `<small>`?